portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -62,23 +74,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
+    List<Comment> comments = new ArrayList<>();
     List<String> messages = new ArrayList<>();
```

Comment:
I'm not really clear on what the purpose of messages is?

Reply:
The purpose of messages is to have a List<String> to use to get all the comments translated in one RPC call, while comments is used to pair each comment and its imageURL together as a json object.

Reply:
Got it. Can you add that as one concise line of documentation above comments and one above messages?